src/pages/index.js

```diff
@@ -83,3 +138,28 @@ function enableValidation(config) {
 }
 
 enableValidation(config);
+
+const api = new Api({
+  baseURL: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f88f7f13-d094-4973-ab37-f03c8d3d09a5",
+    "Content-Type": "application/json",
+  },
+});
+
+// Initialize Page from Server:
+api.getPageInfo()
+  .then(([cardArray, userInfo]) => {
+    console.log(cardArray);
+
+    // Update User Details from Server
+    profileImage.src = userInfo.avatar;
+    user.setUserInfo(userInfo.name, userInfo.about);
+    
+    // Update Cards from Server
+    cardSection = new Section(
+      { items: cardArray, renderer: handleCreateCard },
+      ".cards__list"
+    );
+  })
+  .catch(err => console.error(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors